news/2016-08-05-explore-the-qat.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Explore the QAT!"
+permalink: https://osuqat.tumblr.com/post/148493430910/explore-the-qat
```

Comment:
this property does nothing..? why was it added here

Reply:
really? I wasn't aware about that. I'll remove them in 9 hours

Reply:
were they used in the past? I assume u copied this from some earlier template but idk where it came from

Reply:
The last news post that had this was [Beatmap Spotlights: June 2017](https://github.com/ppy/osu-wiki/blob/master/news/2017-08-02-beatmap-spotlight-june-2017.md). All posts before that used it as well, including the ones I ported over in #1311 and #1001. I assume that it was because every news post afterwards were done here, thus the `tumblr_url` and `permalink` values weren't needed anymore.

idk, I think it shouldn't hurt keeping them, but you think they should all be removed?

Reply:
what I'm asking is where the `permalink` property came from in the first place, I can't find anything detailing why it was put here